### PR TITLE
feat(lbaas): add `Scheme` property to `http_redirect` action

### PR DESCRIPTION
**Pull request by kangasta**

### CODE REVIEW

File: upcloud/load_balancer.go [hunk 1]
```diff
@@ -323,9 +327,10 @@ type LoadBalancerActionHTTPReturn struct {
 	Payload     string `json:"payload,omitempty"`
 }
 
-// LoadBalancerActionHTTPRedirect represents 'http_redirect' action
+// LoadBalancerActionHTTPRedirect represents 'http_redirect' action. Only either Location or Scheme should be defined.
```
Comment (villevsv-upcloud) [inline upcloud/load_balancer.go:330]: I was thinking if the SDK itself should enforce only one of "Location" or "Scheme", but perhaps that would be over-engineering. Especially, if other similar cases are currently validated only by the API and not by the SDK.

**Review by peknur — APPROVED** [review]

File: upcloud/request/load_balancer_helpers.go [hunk 2]
```diff
@@ -31,6 +31,15 @@ func NewLoadBalancerHTTPRedirectAction(location string) upcloud.LoadBalancerActi
 	}
 }
 
+func NewLoadBalancerHTTPRedirectSchemeAction(scheme upcloud.LoadBalancerActionHTTPRedirectScheme) upcloud.LoadBalancerAction {
```
Comment (villevsv-upcloud) [inline upcloud/request/load_balancer_helpers.go:34]: In addition to adding this, maybe the `NewLoadBalancerHTTPRedirectAction` could be renamed to `NewLoadBalancerHTTPRedirectLocationAction` for clarity. Would had to be done in a next major release tho, as it's a breaking change.

**Review by villevsv-upcloud — APPROVED** [review]
LGTM, left some non-blocking comments.

File: upcloud/load_balancer.go (hunk 1 above)
Comment (kangasta) [reply upcloud/load_balancer.go:330]: Yeah, I think it is better to keep this part simple and rely on errors given by API.

The validation could be done in TF provider and `upctl`. If those end up very similar we could then move the validation here.

**Pull request merged**